src/components/TreesMap/TreesMap.tsx

```diff
@@ -122,10 +122,15 @@ if (
   `);
 }
 
+const initialLatitude = +process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
```

Comment:
Why the +process.env.VARNAME?

Reply:
This gets the env var value and automatically returns it as number type. Do you prefer a different approach?

Reply:
Nice. I did not know that this works.


```ts
describe("initialLatitude", () => {
	it("should parse string value to a number", () => {
		process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE = "2324";
		const initialLatitude =
			+process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
		expect(initialLatitude).toBe(2324);
	});

	it("should use default value when environment variable is undefined", () => {
		process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE = undefined;
		const initialLatitude =
			+process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
		expect(initialLatitude).toBe(52.500869);
	});

	it("should use default value when environment variable is null", () => {
		process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE = null;
		const initialLatitude =
			+process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
		expect(initialLatitude).toBe(52.500869);
	});

	it("should parse number value to a number", () => {
		process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE = "123";
		const initialLatitude =
			+process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
		expect(initialLatitude).toBe(123);
	});

	it("should use default value when environment variable cannot be parsed to a number", () => {
		process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE = "invalid";
		const initialLatitude =
			+process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE || 52.500869;
		expect(initialLatitude).toBe(52.500869);
	});
});
```


Only thing is the Typescript compiler throws an error.

```text
error TS18048: 'process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE' is possibly 'undefined'.
```

So something more explicit would be better.

```ts
const initialLatitude =
	!process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE ||
	isNaN(parseFloat(process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE))
		? 52.500869
		: parseFloat(process.env.NEXT_PUBLIC_MAP_INITIAL_LATITUDE);

console.log(initialLatitude);
```